doc/sphinx/changes.rst

```diff
@@ -1224,6 +1224,13 @@ Changes in 8.12.1
   <https://github.com/coq/coq/pull/12738>`_, fixes `#7015
   <https://github.com/coq/coq/issues/7015>`_, by Gaëtan Gilbert).
 
+- **Fixed:**
+  Polymorphic side-effects inside monomorphic definitions were incorrectly
+  handled as not inlined. This allowed to derive an unsoundness
```

Comment:
```suggestion
  handled as not inlined. This allowed deriving an unsound result
```

result? inference? not sure what word you'd use

Reply:
The first change is indeed more idiomatic, but "unsoundness" is the standard terminology AFAICT.

Reply:
"This allowed to prove False"

Reply:
Funnily enough that's the first thing I wrote before pushing the PR but I found it not precise enough. `False` is not a built-in type, you could define it the way you want.

Reply:
"unsoundness" is a binary quality, right?, so "an unsoundness" is incorrect.  If you mean to use "unsoundness" to mean something like "a specific proof of False", that sounds terrible.

> `False` is not a built-in type, you could define it the way you want.

That would be an unusual corner case.  The common understanding would be that `False` is the empty type as usual.

Reply:
"derive an unsoundness" sounds wrong (to this native English speaker), "derive/prove unsoundness" (no article) sounds better, but a bit stuffy. (Or maybe "This caused the unsoundness of Coq".)

I'd suggest "This made Coq unsound." if you want to avoid arguments over "False", though gares's suggestion sounds perfectly fine to me.

Reply:
@ppedrot Proofs of False is what we have been consistently saying over the years. If you do not want to say this, you may use "inconsistency" instead of "unsoundness" (this would probably sound better to native English speakers) and it is anyway more specific (I think). Our kind tag is "kind: inconsistency" with "Proof of False accepted by kernel and/or checker" as a description.